sources/osgShadow/shaders/shadowLinearSoft.glsl

```diff
@@ -1,36 +1,48 @@
 #pragma include "floatFromTex.glsl"
 
-
-#ifdef _OUT_DISTANCE
-#define OPT_ARG_outDistance ,out float outDistance
-#else
-#define OPT_ARG_outDistance
-#endif
-
 // simulation of texture2Dshadow glsl call on HW
 // http://codeflow.org/entries/2013/feb/15/soft-shadow-mapping/
-float texture2DCompare(const in sampler2D depths, const in vec2 uv, const in float compare, const in vec4 clampDimension){
+float texture2DCompare(const in sampler2D depths,
+                       const in vec2 uv,
+                       const in float compare,
+                       const in vec4 clampDimension){
     float depth = getSingleFloatFromTex(depths, clamp(uv, clampDimension.xy, clampDimension.zw));
     return compare - depth;
 }
 
+#ifdef _JITTER_OFFSET
+
+#define INT_SCALE3 vec3(.1031, .1030, .0973)
+#define _rand3out(_p) _p += dot(_p, _p.yzx + 19.19); return fract((_p.xxy + _p.yzz) * _p.zyx);
+vec3 randJitter(vec2 p2) { vec3 p3  = fract(vec3(p2.xyx) * INT_SCALE3); _rand3out(p3) }
+
+#endif //_JITTER_OFFSET
+
 // simulates linear fetch like texture2d shadow
 float texture2DShadowLerp(
     const in sampler2D depths,
     const in vec4 size,
     const in vec2 uv,
     const in float compare,
     const in vec4 clampDimension
-    OPT_ARG_outDistance){
+    OPT_ARG_outDistance
+    OPT_ARG_jitter){
 
     vec2 f = fract(uv * size.xy + 0.5);
+    vec2 centroidCoord = uv * size.xy + 0.5;
     vec2 centroidUV = floor(uv * size.xy + 0.5) * size.zw;
 
+#ifdef _JITTER_OFFSET
+    centroidUV += jitter * ((size.zw *(((randJitter(centroidCoord.xy + gl_FragCoord.xy).xy) - 0.5))) * 0.5);
+#endif //_JITTER_OFFSET
+
+    const vec2 shift  = vec2(  1.0, 0.0);
```

Comment:
Why so many spaces?
(2 between shift and equal instead of 1)
(2 in the vec2 instead of 0)